src/number_theory/bernoulli.lean

```diff
@@ -257,3 +257,120 @@ begin
   { ext x, rw bernoulli_eq_bernoulli' (succ_ne_zero x ∘ succ.inj) },
   { ring },
 end
+
+
+theorem bernoulli_power_series:
```

Comment:
This is the other power series for bernoulli numbers. I have a proof ready, but I think @laughinggas also had pull request for that?! Perhaps it is cleanest to use `bernoulli'_power_series` in the proof here.

Reply:
Yes, I had a proof using `bernoulli'_power_series`, but it requires a bit of cleanup etc etc, so pls feel free to go ahead with yours. I also think the best way to do it would be to apply `eval_neg_hom` (which is injective, to be proved), and then use `bernoulli'_power_series`.

Reply:
I plan to open a separate pull request for `bernoulli_power_series` and put it as a dependency here.